pkg/vpcmodel/externalNetwork.go

```diff
@@ -99,6 +99,13 @@ func newExternalNode(isPublicInternet bool, ipb *common.IPBlock) (Node, error) {
 		isPublicInternet: isPublicInternet}, nil
 }
 
+func NewExternalNodeForCidr(cidr string) Node {
```

Comment:
same here about export?

Reply:
This one is in fact used in `resizeNodes` in `semanticDiffSubnets.go`

Reply:
but it's in the same pkg..

Reply:
right... fixed